commands/FBPrintCommands.py

```diff
@@ -20,6 +20,7 @@
 def lldbcommands():
   return [
     FBPrintViewHierarchyCommand(),
+    FBPrintWindow(),
```

Comment:
Now that `pviews` supports `-w`, I don't think it's worth having a separate command. A user could get `pwindows` by adding this to their `~/.lldbinit`:

```
command alias pwindow pviews --window %1
```

Reply:
Yup, I tried to delete it but it came back in the merge somehow lol, I'm too used to hg these days. I'll remove it